modules/search/instant-search/components/overlay.scss

```diff
@@ -4,16 +4,17 @@
 	width: 100vw;
 	height: 100vh;
 	opacity: 0.975;
-	transition: opacity 0.5s linear, 0.25s transform ease-in-out;
+	transition: opacity 0.5s linear, 0.5s transform ease-in-out;
 	position: fixed;
 	background: white;
 	padding: 3em 1em;
 	overflow-y: auto;
 	overflow-x: hidden;
 	z-index: 9999999999999;
+	animation-fill-mode: forwards;
 
 	&.is-hidden {
-		transform: translateY( -100vh );
+		transform: translateX( 200vw );
```

Comment:
I do prefer having the animation from the side!

This could be changed to `transform: translateX( 100vw );` which makes the transition faster since it only has to move 1 screen's width instead of two.

Reply:
Interestingly I found that `100vw` only moved the overlay halfway back to the right (tested in Firefox):

<img width="1458" alt="Screen Shot 2020-01-15 at 14 17 46" src="https://user-images.githubusercontent.com/17325/72396569-f5b89b00-37a1-11ea-927f-9293fb73fd02.png">

Reply:
That's weird... I'll do some testing this morning to debug that.